src/core/lib/iomgr/buffer_list.cc

```diff
@@ -55,10 +55,16 @@ void fill_gpr_from_timestamp(gpr_timespec* gts, const struct timespec* ts) {
   gts->clock_type = GPR_CLOCK_REALTIME;
 }
 
+void default_timestamps_callback(void* arg, grpc_core::Timestamps* ts,
+                                 grpc_error* shudown_err) {
+  gpr_log(GPR_DEBUG, "Timestamps callback has not been registered");
```

Comment:
Should this be error log or is it too spammy?

You could also make it a log once function:

```
static bool logged = false;
if (!logger) gpr_log(...);
logged = true;
```

Reply:
Got it